packages/manager/apps/pci-kubernetes/src/components/service/ClusterInformation.component.tsx

```diff
@@ -0,0 +1,145 @@
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+  ODS_TILE_VARIANT,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsClipboard,
+  OsdsDivider,
+  OsdsText,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { TKube } from '@/types';
+import ClusterStatus from './ClusterStatus.component';
+import TileLine from './TileLine.component';
+
+export type ClusterInformationProps = {
+  kubeDetail: TKube;
+};
+
+export default function ClusterInformation({
+  kubeDetail,
+}: ClusterInformationProps) {
+  const { t } = useTranslation('service');
+  const { t: tDetail } = useTranslation('listing');
+  const { t: tKubernetes } = useTranslation('kubernetes');
+
+  const ip = () => kubeDetail?.privateNetworkConfiguration?.defaultVrackGateway;
+  const publicNetwork = () =>
+    !kubeDetail?.privateNetworkConfiguration ||
+    !kubeDetail?.privateNetworkConfiguration?.privateNetworkRoutingAsDefault;
+  const privateNetwork = () =>
+    kubeDetail?.privateNetworkConfiguration?.privateNetworkRoutingAsDefault;
```

Comment:
there is no need to define those as functions since there are no parameters

you can just do

```
const ip = kubeDetail?.privateNetworkConfiguration?.defaultVrackGateway;
```

ect

Reply:
done